LetsBake/Timer/View/TimerViewController.swift

```diff
@@ -98,119 +94,140 @@ class TimerViewController: UIViewController {
-      pauseTimerButton.isEnabled = true
+      pauseTimerButton.isEnabled = false
       timer?.invalidate()
     }
   }
-
-  @objc func excuteTimePicker(_ sender: Any) {
-    print("timer")
-  }
-
+  
   @objc func clickedStartTimeButton(_ sender: UIButton) {
-    timePicker.reloadInputViews()
-    pauseTimerButton.isEnabled = true
-    stopTimerButton.isEnabled = true
-    startTimerButton.isEnabled = false
-    timePicker.isHidden = true
-
-    if timer == nil {
-      timeCount = (inputHour * 3600) + (inputMinute * 60) + inputSecond
+    if timeCount != 0 {
+      UIView.animate(withDuration: 0.5, animations: {
+        self.buttonStackView.transform = CGAffineTransform(translationX: 0, y: -100)
+        self.timeLabel.transform = CGAffineTransform(translationX: 0, y: 50)
+      })
+      timePicker.reloadInputViews()
+      pauseTimerButton.isEnabled = true
+      stopTimerButton.isEnabled = true
+      startTimerButton.isEnabled = false
+      timePicker.isHidden = true
+      
+      if timer == nil {
+        timeCount = timerViewModel?.setTimeCount(hour: inputHour, minute: inputMinute, second: inputSecond) ?? 0
+      }
+      timer = Timer.scheduledTimer(timeInterval: 1, target: self, selector: #selector(excuteTimeCount), userInfo: nil, repeats: true)
     }
-
-    timer = Timer.scheduledTimer(timeInterval: 1, target: self, selector: #selector(excuteTimeCount), userInfo: nil, repeats: true)
   }
-
+  
   @objc func clickedPauseTimeButton(_ sender: UIButton) {
     pauseTimerButton.isEnabled = false
     startTimerButton.isEnabled = true
     stopTimerButton.isEnabled = true
     timer?.invalidate()
   }
-
+  
   @objc func clickedStopTimeButton(_ sender: UIButton) {
+    UIView.animate(withDuration: 0.5, animations: {
+      self.buttonStackView.transform = CGAffineTransform(translationX: 0, y: 0)
+      self.timeLabel.transform = CGAffineTransform(translationX: 0, y: 0)
+    })
     stopTimerButton.isEnabled = false
     startTimerButton.isEnabled = true
-    pauseTimerButton.isEnabled = true
+    pauseTimerButton.isEnabled = false
     timePicker.isHidden = false
     timer?.invalidate()
-    inputHour = 0
-    inputSecond = 0
-    inputMinute = 0
-    timeLabel.text = timerViewModel.setTimeLabel(hour: 0, minute: 0, second: 0)
+    
     timePicker.selectRow(0, inComponent: 0, animated: false)
     timePicker.selectRow(0, inComponent: 1, animated: false)
     timePicker.selectRow(0, inComponent: 2, animated: false)
+    inputHour = 0
+    inputMinute = 0
+    inputSecond = 0
+    timeCount = timerViewModel?.setTimeCount(hour: 0, minute: 0, second: 0) ?? 0
+    timerViewModel?.setTimeLabel(timeCount: timeCount)
   }
 }
 
 extension TimerViewController: UIPickerViewDelegate, UIPickerViewDataSource {
+  func configPickerView() {
+    timePicker.subviews[0].backgroundColor = UIColor.white
+    timePicker.subviews[1].backgroundColor = UIColor.white
+    timePicker.subviews[2].backgroundColor = UIColor.white
```

Comment:
```suggestion
    timerPicker.subviews.forEach { $0.backgroundColor = .white } 
```
요런식의 고차함수도 사용 가능 하겠네요!!

Reply:
안그래도 toString 메소드의 경우에 위치가 어디가 좋을지 고민하다 모델에 두었는데요, 상원님 의견은 어떠세요?